--- bin/cassandra.in.sh.orig	2014-01-11 22:55:26.351126308 +0530
+++ bin/cassandra.in.sh	2014-01-12 00:59:18.600041865 +0530
@@ -15,12 +15,12 @@
 # limitations under the License.
 
 if [ "x$CASSANDRA_HOME" = "x" ]; then
-    CASSANDRA_HOME="`dirname "$0"`/.."
+    CASSANDRA_HOME="/usr/share/cassandra"
 fi
 
 # The directory where Cassandra's configs live (required)
 if [ "x$CASSANDRA_CONF" = "x" ]; then
-    CASSANDRA_CONF="$CASSANDRA_HOME/conf"
+    CASSANDRA_CONF="/etc/cassandra/"
 fi
 
 # This can be the path to a jar file, or a directory containing the 
@@ -44,5 +44,5 @@
 if [ "$JVM_VENDOR" != "OpenJDK" -o "$JVM_VERSION" \> "1.6.0" ] \
       || [ "$JVM_VERSION" = "1.6.0" -a "$JVM_PATCH_VERSION" -ge 23 ]
 then
-    JAVA_AGENT="$JAVA_AGENT -javaagent:$CASSANDRA_HOME/lib/jamm-0.2.5.jar"
-fi
\ No newline at end of file
+    JAVA_AGENT="$JAVA_AGENT -javaagent:$CASSANDRA_HOME/jamm-0.2.5.jar"
+fi
